cmd/installer/subcommands/installer/command.go

```diff
@@ -85,6 +139,24 @@ func newSpan(operationName string) (ddtrace.Span, context.Context) {
 	return tracer.StartSpanFromContext(context.Background(), operationName, spanOptions...)
 }
 
+func bootstrapCommand() *cobra.Command {
+	var timeout time.Duration
+	cmd := &cobra.Command{
+		Use:     "bootstrap",
+		Short:   "Bootstraps the package with the first version.",
+		GroupID: "bootstrap",
+		RunE: func(_ *cobra.Command, _ []string) (err error) {
+			b := newBootstraperCmd("bootstrap")
+			defer func() { b.Stop(err) }()
+			ctx, cancel := context.WithTimeout(b.ctx, timeout)
+			defer cancel()
+			return bootstraper.Bootstrap(ctx, b.opts...)
+		},
+	}
+	cmd.Flags().DurationVarP(&timeout, "timeout", "T", 3*time.Minute, "timeout to bootstrap with")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Should we enforce a maximum?

Reply:
If someone wants to footgun themselves it's fine we can allow it :trollface: